js/commonMenu.js

```diff
@@ -314,90 +306,3 @@ module.exports.aboutBraveMenuItem = () => {
     }
   }
 }
-
-module.exports.buildBraveryMenu = function (settings, init) {
```

Comment:
especially since there is actually no way to get to a site that is blocked by safebrowsing using per-site Bravery settings (since the blocked page redirects immediately to `about:safebrowsing`).